docs/DataWorkflows/FetchingYourData.md

```diff
@@ -143,8 +141,7 @@ sub-colornest002
 sub-colornest195
 ```
 
-This directory contains all the outputs from xcp_d for each subject. Now, let's see if we can find the
-actual file we want to get for each subject. We can see all the files for a single subject with `find`
+This directory contains all the outputs from xcp_d for each subject. Now, let's see if we can find the actual file we want to get for each subject. We can see all the files for a single subject with `find`
 
 ```
 $ find sub-colornest001
```

Comment:
I'd change this sentence: 

Old: You may be asked for your password if you haven't [set up an ssh key](#set

New: "You will be asked for your password."

Reply:
Then I'd add a new subsection: 
I see that the atlas I want (Gordon) can be found for each subject with a pattern. 

### Getting files for the entire dataset
To actually transfer these files for all participants, I need to tell datalad to fetch them from PMACs. This is done using wildcards (*) with the following command: